fabric-renderer-indigo/src/main/java/net/fabricmc/fabric/impl/client/indigo/renderer/mesh/QuadViewImpl.java

```diff
@@ -163,7 +163,8 @@ public void copyTo(MutableQuadView target) {
 
 		final MutableQuadViewImpl quad = (MutableQuadViewImpl) target;
 		// copy everything except the material
-		System.arraycopy(data, baseIndex + 1, quad.data, quad.baseIndex + 1, EncodingFormat.TOTAL_STRIDE - 1);
+		System.arraycopy(data, baseIndex, quad.data, quad.baseIndex, EncodingFormat.TOTAL_STRIDE);
+		quad.data[quad.baseIndex + HEADER_BITS] = EncodingFormat.clearMaterial(quad.data[quad.baseIndex + HEADER_BITS]);
```

Comment:
Should this preserve the existing material stored in the `MutableQuadViewImpl` rather than clearing it?

Reply:
The javadoc for this method says that everything except the material is copied.